rest/api_collections_test.go

```diff
@@ -419,15 +419,26 @@ func TestCollectionsChangeConfigScope(t *testing.T) {
 	tb := base.GetTestBucket(t)
 	defer tb.Close()
 	ctx := base.TestCtx(t)
-	err := base.CreateBucketScopesAndCollections(ctx, tb.BucketSpec, map[string][]string{
+
+	scopesAndCollections := map[string][]string{
 		"fooScope": {
 			"bar",
 		},
 		"quxScope": {
 			"quux",
 		},
-	})
+	}
+	err := base.CreateBucketScopesAndCollections(ctx, tb.BucketSpec, scopesAndCollections)
 	require.NoError(t, err)
+	defer func() {
+		collection, err := base.AsCollection(tb.DefaultDataStore())
+		assert.NoError(t, err)
```

Comment:
nitpick but possibly avoid panic if we get here without it being a collection?

```suggestion
		require.NoError(t, err)
```

Reply:
you usually don't want to do `require` inside a defer, but this might make sense here